### PR TITLE
#1881 Slider adjustments + placeholder

### PR DESCRIPTION
**Pull request by riha112**
In this PR:
* Removed pagination in mobile
* Made Slider full-screen in mobile & tablet

Placeholder
* Placeholder for slider is showing, but due to slow internet images that are displayed may be outputted after widget is loaded.

### CODE REVIEW

File: src/app/component/SliderWidget/SliderWidget.component.js [hunk 1]
```diff
@@ -135,10 +137,18 @@ export class SliderWidget extends PureComponent {
         );
     };
 
+    renderPlaceholder() {
+        return <div block="WidgetFactory" elem="SliderPlaceholder" />;
+    }
```
Comment (yeegor) [inline src/app/component/SliderWidget/SliderWidget.component.js:142]: Why is this necessary here? This div does not have any styles, why not just `return null`?

Comment (riha112) [reply src/app/component/SliderWidget/SliderWidget.component.js]: Reverted, back to `null`... due to bad local environment `return null` was not loading... after reinstall all is working.
